util/reggen/ip_block.py

```diff
@@ -80,18 +121,28 @@
     'expose_reg_if': ['pb', 'if set, expose reg interface in reg2hw signal'],
     'interrupt_list': ['lnw', "list of peripheral interrupts"],
     'inter_signal_list': ['l', "list of inter-module signals"],
-    'no_auto_alert_regs': [
+    'no_auto_id_regs': [
         's', "Set to true to suppress automatic "
-        "generation of alert test registers. "
-        "Defaults to true if no alert_list is present. "
-        "Otherwise this defaults to false."
+        "generation of CIP_ID and version registers. "
+        "Defaults to true."
+    ],
+    'no_auto_feat_regs': [
+        's', "Set to true to suppress automatic "
+        "generation of parameter block containing the feature registers. "
+        "Defaults to true."
```

Comment:
To be hones I don't fully understand the involved Python logic here and below but IMHO the default should be that these registers are generated. (For interrupts and alerts we also generate these regs unless the list of interrupts/alerts is empty.)

Reply:
The registers are generated by default - I added this switch for experimentation, special IPs or for cases where the regtool is used outside of a CIP_ID context. For example, I currently add these switches to AST because the internal version at Nuvoton does not yet have these registers.

Reply:
Ah I see, `Defaults to true.` is a wrong description, it should be `Defaults to false.`. Will fix.

Reply:
Fixed.